tests/integration_tests.rs

```diff
@@ -43,9 +46,16 @@ impl AmbitTester {
         self
     }
 
+    // Creates a custom file in home_path directory. Mimic host_file.
+    fn with_host_file(self, name: &str) -> Self {
+        File::create(self.host_path.join(name)).unwrap();
+        self
+    }
+
     // Creates configuration file and repository directory with .git.
     fn with_default_paths(self) -> Self {
         fs::create_dir_all(&self.repo_path.join(".git")).unwrap();
+        fs::create_dir_all(&self.host_path).unwrap();
```

Comment:
`host_path` is already equal to `temp_dir` which is already created when `AmbitTester` is instantiated. This is unneeded.

Additionally, line `57` `create_dir_all` would already create all parent directories (which includes `host_path`)...